README.md

```diff
@@ -101,9 +103,9 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
-[] `$(".post").html()`
+[x] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").innerHTML`
+[x] `$(".post").html()`
 [] `document.getElementsByClassName("post")[0].innerHTML`
 [] `document.getElementsByClassName("post").innerHTML`
 ```
```

Comment:
Two of your three selections are correct, but `$(".post").innerHTML` is an invalid combination of jQuery and vanilla Javascript.  `document.getElementsByClassName("post")[0].innerHTML` is also a correct response.